src-json/meta.json

```diff
@@ -614,6 +620,13 @@
 		"targets": ["TClass", "TEnum"],
 		"internal": true
 	},
+	{
+		"name": "JvmDefault",
+		"metadata": ":jvm.default",
```

Comment:
Meta name is not clear.
Maybe `@:jvm.hasDefaultImplementation`?

Reply:
The Java API just uses the `default` keyword for such methods, so in that context it should be quite clear.